src/smsCreditCalc.ts

```diff
@@ -1,25 +1,27 @@
-const utf8Characters = new RegExp(
+// Regex for GSM 03.38 7bit character set
+// Source: https://frightanic.com/software-development/regex-for-gsm-03-38-7bit-character-set/
```

Comment:
It might be worth copying the comment that explains the regular expression, in case that website goes down.

Reply:
I checked and this site is archived on the Wayback machine. So even if the site goes down we could retrieve the original comment.